Source/Scene/processModelMaterialsCommon.js

```diff
@@ -643,7 +643,7 @@ function generateTechnique(
     fragmentShader += "  vec3 normal = normalize(v_normal);\n";
     if (khrMaterialsCommon.doubleSided) {
       // !gl_FrontFacing doesn't work as expected on Mac/Intel so use the more verbose form instead. See https://github.com/CesiumGS/cesium/pull/8494.
-      fragmentShader += "  if (gl_FrontFacing == false)\n";
+      fragmentShader += "  if (czm_backFacing())\n";
```

Comment:
Should the comment above be removed now that it's in the czm_backFacing file? Same for wherever else it occurs.